tests/test_utils.py

```diff
@@ -202,6 +202,46 @@ def test_get_custom_task():
     assert "Task not found" in str(f)
 
 
+def test_prep_problem():
+    """Tests prep_problem().
+
+    Note: I separately ran prep_problem() and then planned with the
+    new problem. Pyperplan still planned correctly with it.
+    """
+    task01_path = utils.get_custom_task('dressed', 1).problem_file
```

Comment:
nice test! for a function like this with a bunch of tricky edge cases, I think we should write a few more tests. 

can you write some tests that just involve a manually constructed string input to the new function? (so, it doesn't have to come from an actual PDDL file, and it doesn't even necessarily need to be an entire PDDL problem). try to think of tricky edge cases.